tests/taskcluster/bin/raptor_graph

```diff
@@ -23,6 +23,9 @@ var originalHeadRev = process.env.GITHUB_HEAD_REV;
 
 const TC_CLIENT_OPTS = { timeout: 30 * 1000 };
 
+// taskcluster worker type for raptor tasks
+RAPTOR_WORKER_TYPE = 'raptor-single-capacity';
```

Comment:
That's a good point, so the raptor decision task itself runs on 'gaia-decision', but right the results task can probably just run on 'raptor' and not on 'raptor-single-capacity'. Only problem is, then the results task would show up on treeherder on a different line than the other raptor tasks... hmm.. but something to definitely consider optimizing in the future, thanks!